modules/ROOT/pages/planning-and-tuning/runtimes/concepts.adoc

```diff
@@ -239,7 +239,7 @@ However, for most queries, the pipelined runtime is a more efficient runtime cap
 == Parallel runtime
 
 Both the slotted and pipelined runtimes execute queries in a single thread assigned to one CPU core.
-It is still possible to achieve parallelism (broadly defined as when two or more sets of operations can be processed concurrently within a single database environment) when using these two runtimes by running multiple queries in separate CPU threads concurrently.
+It is still possible to achieve parallelism (broadly defined as when two or more sets of operations can be processed concurrently within a single database environment) when using these two runtimes by running multiple queries in separate CPU threads concurrently (for examples, see xref:subqueries/subqueries-in-transactions.adoc#concurrent-transactions[`CALL` subqueries in transactions -> Concurrent transactions]).
```

Comment:
CALL IN CONCURRENT TRANSACTIONS is not really an example of what is meant here, but rather a new additional way to achieve concurrency. I would suggest using a separate sentence for this; something like:

"Another way is to run multiple transactions concurrently within the same query using CALL {...} IN CONCURRENT TRANSACTIONS (for examples, see xref:subqueries/subqueries-in-transactions.adoc#concurrent-transactions[`CALL` subqueries in transactions -> Concurrent transactions])."